src/main/resources/org/broadinstitute/dsde/sam/liquibase/changesets/initial_schema.xml

```diff
@@ -63,6 +63,9 @@
             <column name="google_subject_id" type="VARCHAR(30)">
                 <constraints unique="true" nullable="false"/>
             </column>
+            <column name="display_name" type="VARCHAR(100)">
```

Comment:
varchar(100) strategically chosen because that's the google-defined limit for display names